app/models/rental.py

```diff
@@ -0,0 +1,26 @@
+from flask import current_app
+from sqlalchemy.sql.schema import ForeignKey
+from app import db
+from sqlalchemy.orm import relationship
+from datetime import datetime
+# from sqlalchemy import Table, Column, Integer, ForeignKey # Need?
+
+class Rental(db.Model):
+    __tablename__ = "rentals"
+
+    rental_id =db.Column(db.Integer, primary_key=True, autoincrement=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('videos.video_id'),nullable=True)
+    customer_id = db.Column(db.Integer, ForeignKey("customers.customer_id"),primary_key=True)
+    due_date = db.Column(db.DateTime, nullable=True)
+    customer_id = db.Column(db.Integer, ForeignKey("customers.customer_id"),primary_key=True)
+    
```

Comment:
You may want to add a check in date to know if the user returned this rental:

```suggestion
    check_in_date = db.Column(db.DateTime, nullable=True)
```